src/tests/component/signup.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Signup } from '../../components/Auth/Signup';
+
+const props = {
+  setSignupInput: jest.fn(),
+  errors: null
+};
+describe('<Signup />', () => {
+  test('should render <Signup />', () => {
+    const wrapper = shallow(<Signup {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('should render <Signup />', () => {
+    const event = {
+      preventDefault: jest.fn(),
+    };
+    const wrapper = shallow(<Signup {...props} />);
+    const { handleSubmit } = wrapper.instance();
+    handleSubmit(event);
+    expect(event.preventDefault).toHaveBeenCalled();
+    expect(props.setSignupInput).toHaveBeenCalled();
+  });
+
+  test('should render <Signup />', () => {
+    const event = {
+      preventDefault: jest.fn(),
+      target: {
+        value: '123456',
+        name: 'password',
+      }
+    };
+    const newProps = { ...props, errors: 'user not found ' };
+    const wrapper = shallow(<Signup {...newProps} />);
+    const { handleInput } = wrapper.instance();
+    handleInput(event);
+    expect(wrapper.state().password).toEqual(event.target.value);
+  });
+});
```

Comment:
Declare the wrapper outside or in a before block so that you do not have to declare it in each test suit.
Also, all the test suits in this file have the same description should render auth please change it.